python/pyspark/sql/dataframe.py

```diff
@@ -1393,6 +1393,16 @@ def replace(self, to_replace, value=None, subset=None):
         |null|  null| null|
         +----+------+-----+
 
+        >>> df4.na.replace('Alice', None).show()
```

Comment:
Actually, I think this should be something to be fixed in `DataFrameNaFunctions.replace` in this file ...